src/django_pg_migration_tools/operations.py

```diff
@@ -192,6 +192,9 @@ class SaferRemoveIndexConcurrently(
     model_name: str
     name: str
 
+    def __init__(self, model_name: str, name: str) -> None:
+        return super().__init__(model_name=model_name, name=name)
+
```

Comment:
That's an interesting one. Which IDE are you using?

It may be a bug on the IDE - or it may be caching files locally. There's no significant difference between `SaferRemoveIndexConcurrently` and `SaferAddIndexConcurrently` innit methods.

Keen to see if there's a fix for your IDE instead, so that we don't need to maintain code for a specific IDE in the repo.

Reply:
I'm using Visual Studio Code - will investigate. I'll make a new PR with changes separated so they can be merged easily if they are good to go.